app/controllers/requests_controller.rb

```diff
@@ -15,11 +17,27 @@ def create
       redirect_to root_url
     else
       flash[:danger] = t ".fail"
-
       render :new
     end
   end
 
+  def index
+    @pagy, @requests = pagy Request.recent_post
+  end
+
+  def show; end
+
+  def update
+    if [:fresh, :comfirmed].include?(@request.status.to_sym) &&
+       @request.update(status: params[:request][:status])
```

Comment:
indent

Reply:
Em để 2 điều kiện cùng indent rồi mà chị :((

Reply:
sai indent ấy chớ em
```suggestion
      @request.update(status: params[:request][:status])
```